pages/decadal.vue

```diff
@@ -1,36 +1,69 @@
 <template>
-  <div class="h-full w-full flex flex-wrap gap-4 m-8">
-    <div
-      v-for="dataset in datasets"
-      :key="dataset.title"
-      class="border-4 p-4 prose"
-    >
-      <h2>{{ dataset.title }}</h2>
-      <ul>
-        <li>Contact: <span v-for="person in dataset.contact" :key="person.name"><a target="blank" :href="getOrcid(person)">{{ person.name }}</a>, </span></li>
-        <li>License: {{ dataset.license }}</li>
-        <li>Format: {{ dataset.format }}</li>
-        <li>Data access: <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a></li>
-      </ul>
-      <h3>Description</h3>
-      <nuxt-content :document="dataset" />
+  <div class="h-full w-full">
+    <div class="flex bg-gray-100 m-8">
+      <button
+        v-for="(tag, i) in tabs"
+        :key="i"
+        class="text-gray-600 py-4 px-6 block hover:text-blue-500 focus:outline-none"
+        :class="tag.isActive ? 'text-blue-500 border-b-2 font-medium border-blue-500' : ''"
+        @click="toggle(i)"
+      >
+        {{ tag.tag }}
+      </button>
+    </div>
+    <div class="flex gap-8 m-8">
+      <div class="flex flex-col gap-4 min-w-2/3">
+        <h1 class="text-xl">
+          {{ tab.title }}
+        </h1>
+        <nuxt-content class="break-words prose" :document="tab" />
+        <h1 class="text-xl">
+          List of datasets
+        </h1>
+        <ul
+          v-for="(dataset, i) in tab.datasets"
+          :key="i"
+        >
+          <li>{{ dataset.title }}</li>
+          <li>Contact: {{ dataset.contact[0].name }}</li>
+          <li>Data access:</li>
+          <li class="break-words italic underline">
+            <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a>
+          </li>
```

Comment:
```suggestion
          <li class="prose">DOI: <a :href="dataset.doi" target="blank">{{ dataset.doi }}</a></li>
```
`prose` class also does link formatting, and makes it consistent with the rest of the layout.

Perhaps we can exclude the "https://www.doi/org/" part of the displayed doi (only include it in the link target).

Reply:
Corrected! And the "https://www.doi/org/" is not displayed.